### PR TITLE
Character Display task added

### PR DESCRIPTION
**Pull request by krushna-dongre**
The task is to display any Character on the screen using CSS and HTML
I have create Character D

### CODE REVIEW

File: CharacterDisplay/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html5>
```
Comment (Ramkrish1214) [inline CharacterDisplay/index.html]: Please check the doctype, for html 5 it should be <!doctype html> but not <!doctype html5>

Comment (Ramkrish1214) [reply CharacterDisplay/index.html]: And verify in all html pages

**Review by Ramkrish1214 — COMMENTED** [review]
Please leave your reply in code review comments, once the changes are done and please ping me team common channel

File: CharacterDisplay/index.html (hunk 1 above)
Comment (krushna-dongre) [reply CharacterDisplay/index.html]: changes are done

**Pull request merged**